cnf-certification-test/suite_test.go

```diff
@@ -203,6 +204,12 @@ func TestTest(t *testing.T) {
 	claimOutputFile := filepath.Join(*claimPath, results.ClaimFileName)
 	claimhelper.WriteClaimOutput(claimOutputFile, payload)
 
+	// Send claim file to the collector
```

Comment:
Can this be put behind a config option?  Similar to the section below:

`!configuration.GetTestParameters().OmitArtifactsZipFile`